core/java/android/view/ViewConfiguration.java

```diff
@@ -510,6 +510,15 @@ private ViewConfiguration(Context context) {
                 com.android.internal.R.integer.config_screenshotChordKeyTimeout);
     }
 
+    public static boolean isNumeric(String string) {
```

Comment:
This is just a number verification. the commit title claims to add more. Unless I'm not seeing something?
Do you want to make a config / switch for it?

Reply:
oops looks like you already have this change, my bad. I didn't look at it because it picked without conflicts.
The other change is relevant though.

Reply:
May you force push to this branch with the commit dropped?